components/header.jsx

```diff
@@ -32,6 +29,9 @@ var Header = React.createClass({
               <Link to="sessions">Spaces & Sessions</Link>
             </div>
           </div>
+          <div id="tabzilla">
```

Comment:
I think something about tabzilla is causing horizontal scroll around 800px wide views